bsnes/target-libretro/GNUmakefile

```diff
@@ -1,4 +1,4 @@
-name := bsnes_hd_beta_libretro
+name := bsnes_libretro
```

Comment:
Is it intentional that occurences of bsnes_hd_beta have been renamed to just bsnes?